readme.md

```diff
@@ -8,6 +8,68 @@ Faker requires PHP >= 5.3.3.
 
 [![Monthly Downloads](https://poser.pugx.org/fzaninotto/faker/d/monthly.png)](https://packagist.org/packages/fzaninotto/faker) [![Build Status](https://secure.travis-ci.org/fzaninotto/Faker.png)](http://travis-ci.org/fzaninotto/Faker) [![SensioLabsInsight](https://insight.sensiolabs.com/projects/eceb78a9-38d4-4ad5-8b6b-b52f323e3549/mini.png)](https://insight.sensiolabs.com/projects/eceb78a9-38d4-4ad5-8b6b-b52f323e3549)
 
+# Table of Contents
+
+- [Installation](#installation)
+- [Basic Usage](#basic-usage)
+- [Formatters](#formatters)
+	- [`FakerProviderBase`](#fakerproviderbase)
+	- [`FakerProviderLorem`](#fakerproviderlorem)
+	- [`FakerProvideren_USPerson`](#fakerprovideren_usperson)
+	- [`FakerProvideren_USAddress`](#fakerprovideren_usaddress)
+	- [`FakerProvideren_USPhoneNumber`](#fakerprovideren_usphonenumber)
+	- [`FakerProvideren_USCompany`](#fakerprovideren_uscompany)
+	- [`FakerProvideren_USText`](#fakerprovideren_ustext)
+	- [`FakerProviderDateTime`](#fakerproviderdatetime)
+	- [`FakerProviderInternet`](#fakerproviderinternet)
+	- [`FakerProviderUserAgent`](#fakerprovideruseragent)
+	- [`FakerProviderPayment`](#fakerproviderpayment)
+	- [`FakerProviderColor`](#fakerprovidercolor)
+	- [`FakerProviderFile`](#fakerproviderfile)
+	- [`FakerProviderImage`](#fakerproviderimage)
+	- [`FakerProviderUuid`](#fakerprovideruuid)
+	- [`FakerProviderBarcode`](#fakerproviderbarcode)
+	- [`FakerProviderMiscellaneous`](#fakerprovidermiscellaneous)
+	- [`FakerProviderBiased`](#fakerproviderbiased)
+- [Unique and Optional modifiers](#unique-and-optional-modifiers)
+- [Localization](#localization)
+- [Populating Entities Using an ORM or an ODM](#populating-entities-using-an-orm-or-an-odm)
+- [Seeding the Generator](#seeding-the-generator)
+- [Faker Internals: Understanding Providers](#faker-internals-understanding-providers)
+- [Real Life Usage](#real-life-usage)
+- [Language specific formatters](#language-specific-formatters)
+	- [`FakerProviderat_ATPayment`](#fakerproviderat_atpayment)
```

Comment:
No need to display the subsections here. It will be impossible to keep up to date anyway.